custom-steps/libpng/post-build.ps1

```diff
@@ -0,0 +1,19 @@
+param (
+    [Parameter(Mandatory=$true)][string]$BuildArtifactsPath
+)
+
+Import-Module "$PSScriptRoot/../../ps-modules/Util"
+
+$IsOnMacOS = Get-IsOnMacOS
+
+if(-not $IsOnMacOS) {
+    Write-Host "The step is only required for MacOS.  Skipping step..."
+    exit
+}
+
+Import-Module "$PSScriptRoot/../../ps-modules/MacUtil"
+
+Write-Host "Running post-build script..."
+Push-Location "$BuildArtifactsPath/lib"
```

Comment:
(noted elsewhere) do we need this?

Reply:
Fixed: e5d93971c0d8e3fb07bea1f99163f42b6d52d2e1